cmd/app/createca.go

```diff
@@ -19,22 +19,40 @@ import (
 	"crypto"
 	"crypto/ecdsa"
 	"crypto/rand"
+	"crypto/rsa"
 	"crypto/x509"
 	"crypto/x509/pkix"
 	"encoding/pem"
+	"errors"
 	"fmt"
 	"math"
 	"math/big"
 	"os"
 	"time"
 
 	"github.com/ThalesIgnite/crypto11"
+	"github.com/sigstore/fulcio/pkg/log"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
+)
 
-	"github.com/sigstore/fulcio/pkg/log"
+const (
+	label = "FulcioCA"
```

Comment:
for readability, I might do all caps for constant `LABEL` or even something slightly more specific